cardano-cli/cardano-cli.cabal

```
@@ -288,7 +288,7 @@ library cardano-cli-test-lib
     exceptions,
     filepath,
     hedgehog,
-    hedgehog-extras >=0.6.1,
+    hedgehog-extras >=0.6.1 && <0.6.5.1,
```

Upper bound not needed anymore. You need to update hackage index to a newer than or `2024-12-24T12:56:48Z` though.
```suggestion
    hedgehog-extras >=0.6.1
```